### PR TITLE
add: landing linking and landing sections

### PR DESCRIPTION
**Pull request by lazasa**
- Added routing config for landing (app.routes.ts)
- Landing sections (empty components)
- Navbar
- Footer (empty)

### CODE REVIEW

File: src/app/modules/landing/navbar/navbar.component.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      </li>
+      <li routerLinkActive="text-maroon-1" [routerLinkActiveOptions]="{ exact: true }">
+        <a [routerLink]="'/about'">Sobre nosotros</a>
+      </li>
+      <li routerLinkActive="text-maroon-1" [routerLinkActiveOptions]="{ exact: true }">
+        <a [routerLink]="'/courses'">Cursos</a>
+      </li >
+      <li routerLinkActive="text-maroon-1" [routerLinkActiveOptions]="{ exact: true }">
+        <a [routerLink]="'/contact'">Contactanos</a>
+      </li>
```
Comment (martinalbrnz) [inline src/app/modules/landing/navbar/navbar.component.html:16]: Avoid using hardcoded route values, use an enum instead

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
     {
-        path: "", component: AuthComponent,
-        canActivate: [authGuard]
+        path: "", component: LandingComponent,
+        children: [
+            {path: '', component: HomeComponent},
+            {path: 'about', component: AboutComponent},
+            {path: 'contact', component: ContactComponent},
+            {path: 'courses', component: CoursesComponent}
+        ]
```
Comment (martinalbrnz) [inline src/app/app.routes.ts:21]: Use the lazy import to minimize chunks max size and improve download speed

**Review by martinalbrnz — APPROVED** [review]

**Comment by martinalbrnz** [comment]
Left two obs to apply on next commits/PRs. 
Keep in mind that we'll need internationalization so it's better if we get that soon (I could do it).

File: src/app/modules/landing/navbar/navbar.component.html (hunk 1 above)
Comment (lazasa) [reply src/app/modules/landing/navbar/navbar.component.html:16]: > Avoid using hardcoded route values, use an enum instead

I thought abt it, where'd you put that enum? (like in the same file or ?)

Comment (martinalbrnz) [reply src/app/modules/landing/navbar/navbar.component.html:16]: you've to create a folder to store all constants (think abt localStorage keys, api routes, front routes, etc...) and then create a file for the front end routes. I like to have separated files for api routes, front routes and other constants but you can have everything on a constants.ts file

**Pull request merged**